.env.sample

```diff
@@ -0,0 +1,6 @@
+ENV=demo
```

Comment:
I don't think you need this file.

Reply:
we usually include it so developers know what the env variable are for their local/demo

Reply:
but they're set in the nuxt.config.ts.

Reply:
If there's a reason not to include it, bring it up in the engineering channel and we can decide if we should include these or not as they are on all of our sites. some sites require a local env file, even if the vue/nuxt sites do not I still think it's good to see what the value are since circleci hides them

Reply:
maybe include this in the readme with the rest of the documentation?